### DISCUSSION TITLE
Continuous autocomplete/typeahead?

### DISCUSSION THREAD

**Discussion by suasgn** [digitalfortress-tech/typeahead-standalone#65] [General]
Does typeahead-standalone support a continuous autocomplete/typeahead?

![image](https://github.com/digitalfortress-tech/typeahead-standalone/assets/5080378/825cbdd0-fbca-484d-8a97-15aea203f1f7)

For reference, [ZURB's Tributejs](https://zurb.github.io/tribute/example/) does autocomplete anywhere and continues suggesting when a new word is separated by a space

![image](https://github.com/digitalfortress-tech/typeahead-standalone/assets/5080378/d98725ba-3acb-4cb9-8a75-f5a1d75fa852)

**Comment by niketpathak** [comment]
Hello, currently there is no support for spaced queries. The search by default is designed to return results that are mutual to all input queries. But this seems like a good idea/use case for a future release. Thanks!
